src/06_shared/ui/checkbox/checkbox.scss

```diff
@@ -0,0 +1,53 @@
+.checkbox {
```

Comment:
We are using BEM. Do not style tags

Reply:
It's not a tag. It's a class.

Reply:
You style tags inside the class .checkbox